### DISCUSSION TITLE
Could anyone help with a simple example?

### DISCUSSION THREAD

**Discussion by Hannibal046** [coady/lupyne#12] [Q&A]
Hello,
I am new to search engine and lupyne. And I want to use search engine to help me achieve a simple target which is given a query, I want to search through all documents and return relevant ones in terms of BM25 score? How can I do it?
I tried examples in the doc:
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/38466901/129846289-233575ce-d89f-42bd-bd7c-1fe2031879d8.png">
How can I assign BM25 as scoring function? and should I give different setting like tokenization  when searching different languages ?
Sorry for taking your time! Thanks !

**Comment by coady** [comment]
`IndexSearcher.similarity` can be set, but the default is already `BM25Similarity`.
